apps/resumes/views.py

```
@@ -1,118 +1,67 @@
+        return Resume.objects.filter(user=self.request.user, is_deleted=False)
 
     def perform_destroy(self, instance):
-        # Soft delete
         instance.is_deleted = True
         instance.save()
 
-
 class ResumeAnalyzeView(APIView):
-    """이력서 분석 (AI 기반)"""
+    """이력서 분석 (AI 기반) - 현재는 준비 중 응답"""
     permission_classes = [IsAuthenticated]
```

Ruff (0.14.13)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)